15-foundations-applications.Rmd

```diff
@@ -24,6 +24,32 @@ inference_method_summary_table %>%
   add_header_above(c("", "Answer" = 3))
 ```
 
+You might have noticed that the word *distribution* is used throughout this part (and will continue to be used in future chapters).
+A distribution always describes variability, but sometimes it is worth reflecting on **what** is varying.
+Typically the distribution either describes how the observations vary or how a statistic varies.
+But even when describing how a statistic varies, there is a further consideration with respect to the experimental design (e.g., does the statistic vary from random sample to random sample? or does it vary from random allocation to random allocation?).
+The methods presented in this text (and used in science generally) are typically used interchangeably across ideas of random samples or random allocation of the treatment.
+Oftentimes, two different analysis methods will give equivalent conclusions.
+The most important thing to consider, is how to contextualize the conclusion in terms of the problem.
+See Figure \@ref(fig:randsampValloc) to confirm that your conclusions are appropriate.
+
+Below, we synthesize the different types of distributions discussed throughout the text.
+Reading through the different definitions and solidifying your understanding will help as the distributions are used in future chapters.
+
+::: {.important data-latex=""}
+**Distributions.**
+
+-   A **data distribution** describes the shape, center, and variability of the **observed data**.
+
+-   A **population distribution** describes the shape, center, and variability of the entire **population of data**.
+
+-   A **sampling distribution** describes the shape, center, and variability of all possible values of a **sample statistic** from samples of a given sample size from a given population.
+
+-   A **randomization distribution** describes the shape, center, and variability of all possible value of a **sample statistic** from random allocations of the treatment variable.
+
+-   A **bootstrap distribution** describes the shape, center, and variability of all possible values of a **sample statistic** from resamples of the observed data.
```

Comment:
```suggestion
-   A **bootstrap distribution** describes the shape, center, and variability of all possible values of a **sample statistic** from resamples of the observed data. We computationally generate the bootstrap distribution, though usually, it's not feasible to generate all possible resamples of the observed data, so we instead generate a large number of them.
```

Reply:
Add one more:

Because we are randomly resampling from the observed data, the bootstrap distribution might describe the null hypothesis and it might describe the alternative hypothesis.

Reply:
Hm, I'm a bit torn on this one, because we talk about bootstrapping more in the context of intervals than testing, but this sentence (without an additional one about estimation) could imply bootstrapping is used primarily for testing. How about this instead?

"Since bootstrap distributions are generated by randomly resampling from the observed data, they are centered at the sample statistic. Bootstrap distributions are most often used for estimation, i.e. we base confidence intervals off of them. ​However, for certain parameters where randomization (i.e. reallocation) doesn't make sense, they can also be used for testing."

Thought about also adding "In such scenarios, we often need to shift the bootstrap distribution to be centered at the null hypothesized value of the parameter". Too much?